src/main/java/seedu/duke/logic/commands/gates/CalculateBooleanCommand.java

```diff
@@ -2,16 +2,24 @@
 
 import seedu.duke.DukeException;
 import seedu.duke.model.template.BooleanTemplate;
+import seedu.duke.ui.Ui;
 
 public class CalculateBooleanCommand extends BooleanCommand {
     public static final String COMMAND_WORD = "calc";
+    public static final Ui ui = new Ui();
```

Comment:
This is no longer necessary, along with the import.
```suggestion
```

Reply:
Noted